sys/net/routing/rpl/rpl.c

```diff
@@ -279,32 +273,16 @@ void *rpl_process(void *arg)
 void _rpl_update_routing_table(void)
 {
```

Comment:
Since FIB handles the lifetime of routes, would it make sense to remove this `_rpl_update_routing_table` thread completely and just pass a function callback to the fib, which should be executed when a certain condition holds? (e.g. route lifetime expired) @BytesGalore @OlegHahm